app/views/workshops/show.html.haml

```diff
@@ -13,7 +13,9 @@
     .row
       .medium-12.columns
         %p.lead
-          Attend our workshops to learn programming in a safe and supportive environment at your own pace, or to share your knowledge and coach our students.
+          = t("workshops.lead")
+        %p.description
+          = @workshop.description.html_safe
```

Comment:
@turger brakeman is reporting a `Unescaped model attribute` error on this line. Could we address this before merging?

Reply:
Brakeman is probably right. The description field is a particular problem for XSS issues on Planner. I have a PR sitting on my machine which would covers many issues on the application - I think most of them are descriptions (it's not so bad as a Student can't cause them. It would take a malicious organiser to cause a problem). I was kinda waiting for the PR load to decrease but it seems the opposite is happening.

Reply:
Not sure how I should fix this error? I could remove the possibility to add html to the description or just somehow ignore the error?

Reply:
Remove `html_safe` string property and call `Sanitize` method instead. The difference is that they both allow HTML tags through but `html_safe` allows Javascript and `sanitize` doesn't.

In my fix I've added sanitize to the base class of the presenters so it is a matter of adding a single function call to the output string - it's a bit long otherwise:

`ActionController::Base.helpers.sanitize('<b>bold</b>')`

### Reference
[Sanitize](https://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html) 
[html_safe](https://api.rubyonrails.org/classes/String.html#method-i-html_safe)
[Using sanitize within a Rails controller](https://stackoverflow.com/questions/3985989/using-sanitize-within-a-rails-controller)

Reply:
@turger as @notapatch pointed out your best bet is sanitize, in a haml template you should be able to do `= sanitize(t("workshops.lead") )`.

Reply:
I added sanitize and now brakeman seems to pass!